File: Lib/Telerik/Examples/QuickStart/Forms/TabbedForm/FirstLook/LaunchForm.vb
Imports Telerik.QuickStart.WinControls
Imports Telerik.WinControls.UI

Namespace Telerik.Examples.WinControls.Forms.TabbedForm.FirstLook
    Public Class LaunchForm
        Inherits ExamplesLauncherForm

        Private ReadOnly Property TabbedForm As RadTabbedForm
            Get
                Return TryCast(Me.Form, RadTabbedForm)
            End Get
        End Property

        Public Overrides Sub OnThemeChanged()
            MyBase.OnThemeChanged()

            If Me.TabbedForm IsNot Nothing AndAlso Me.TabbedForm.TabbedFormControl IsNot Nothing Then
                Me.SyncTheme(Me.TabbedForm)
            End If
        End Sub

        Private Sub SyncTheme(ByVal form As RadTabbedForm)
            Me.TabbedForm.ThemeName = Me.CurrentThemeName
            Me.TabbedForm.TabbedFormControl.ThemeName = Me.CurrentThemeName
        End Sub
    End Class
End Namespace

File: Lib/Telerik/Examples/QuickStart/Forms/TabbedForm/FirstLook/LaunchForm.cs
using Telerik.QuickStart.WinControls;
using Telerik.WinControls.UI;

namespace Telerik.Examples.WinControls.Forms.TabbedForm.FirstLook
{
    public class LaunchForm : ExamplesLauncherForm
    {
        private RadTabbedForm TabbedForm
        {
            get { return this.Form as RadTabbedForm; }
        }
        
        public override void OnThemeChanged()
        {
            base.OnThemeChanged();

            if (this.TabbedForm != null && this.TabbedForm.TabbedFormControl != null)
            {
                this.SyncTheme(this.TabbedForm);
            }
        }
        
        private void SyncTheme(RadTabbedForm form)
        {
            this.TabbedForm.ThemeName = this.CurrentThemeName;
            this.TabbedForm.TabbedFormControl.ThemeName = this.CurrentThemeName;
        }
    }
}
